src/devices/bubendorff.ts

```diff
@@ -0,0 +1,12 @@
+import * as m from "../lib/modernExtend";
```

Comment:
Also add this file to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/index.ts